docs/en/tutorials/getting-started.rst

```diff
@@ -1070,22 +1060,17 @@ like this:
 
     echo "Your new Bug Id: ".$bug->getId()."\n";
 
-Since we only have one user and product, probably with the ID of 1, we can call this script with:
+Since we only have one user and product, probably with the ID of 1, we can
+call this script as follows:
 
 ::
 
     php create_bug.php 1 1 1
 
```

Comment:
The statement "This is the first contact with the read API of the EntityManager" is not accurate, as we were previously introduced to EntityManager::find() earlier in the guide.